readme.txt

```diff
@@ -2,6 +2,9 @@
 
 == Changelog ==
 
+= [4.9.4.1] TBD =
```

Comment:
```suggestion
= [4.9.6] TBD =

```

Reply:
Yeah, that version number was from when I thought this was just going to be a style tweak.